dothething/dothething/ViewModels/ClipsViewModel.swift

```diff
@@ -244,9 +246,9 @@ class ClipsViewModel: ObservableObject, ImagePickerMessenger {
                             let thumbnail = Thinger.getThumbnail(url: tempFileUrl, degreesToRotate: self.videoDegreesToRotate)
                             // highlight first clip
                             if self.clips.count == 0 {
-                                self.clips.append(Clip(url: tempFileUrl, thumbnail: thumbnail, isHighlighted: true, metadata: clip))
+                                self.clips.append(Clip(url: tempFileUrl, thumbnail: thumbnail, isHighlighted: true, metadata: clip, showCode: false))
                             } else {
-                                self.clips.append(Clip(url: tempFileUrl, thumbnail: thumbnail, isHighlighted: false, metadata: clip))
+                                self.clips.append(Clip(url: tempFileUrl, thumbnail: thumbnail, isHighlighted: false, metadata: clip, showCode: false))
```

Comment:
The change to set `showCode` to `false` for newly added clips is a sensible default behavior. This ensures that the code metadata is not displayed by default, which aligns with the expected user experience.